src/server.h

```diff
@@ -971,6 +971,7 @@ typedef struct replBufBlock {
 typedef struct dbDictState {
     list *rehashing;                       /* List of dictionaries in this DB that are currently rehashing. */
     int resize_cursor;                     /* Cron job uses this cursor to gradually resize dictionaries (only used for cluster-enabled). */
+    int non_empty_slots;                   /* The number of non-empty slots. */
```

Comment:
but we also have variable names like "slot_size_index" and function names like "getFairRandomSlot" and "dbGetNextNonEmptySlot." Perhaps it would be better to maintain consistency and use "slot" throughout?

Reply:
i agree we wanna use the same term. but i have a feeling it should not be "slot".
well, maybe @guybe7 will make some cleanup for that in https://github.com/redis/redis/pull/12822, and we'll discuss it there.